app_to_web.md

```diff
@@ -48,7 +48,17 @@ The existing API involves sending requests to the reporting origin to register e
 ```
 Attribution-Reporting-Support: os, web
 ```
-If this header indicates OS support, the reporting origin can optionally respond to the request with a [string structured header](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-header-structure-15#section-3.3.3) that indicates a desire to use the OS’s attribution API instead of the browser’s. Note that the API also allows browsers to only support OS-level attribution if they choose.
+
+For subresource requests without the `Attribution-Reporting-Eligible` header,
+the server can optionally respond to the request with a [boolean structured header](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-header-structure-15#section-3-3-6):
+```
+Accept-Attribution-Reporting-Support: ?1
+```
+
+If this header exists and is a true value, the browser will add the
+`Attribution-Reporting-Support` header in all subsequent redirect requests.
+
+If the `Attribution-Reporting-Support` header indicates OS support, the reporting origin can optionally respond to the request with a [string structured header](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-header-structure-15#section-3.3.3) that indicates a desire to use the OS’s attribution API instead of the browser’s. Note that the API also allows browsers to only support OS-level attribution if they choose.
```

Comment:
```suggestion
If the `Attribution-Reporting-Support` header indicates OS support, the reporting origin can optionally respond to the request with a [string structured header](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-header-structure-15#section-3.3.3) that indicates a desire to use the OS's attribution API instead of the browser's. Note that the API also allows browsers to only support OS-level attribution if they choose.
```